src/Symfony/Component/Messenger/Handler/MessageHandlerInterface.php

```diff
@@ -15,6 +15,8 @@
  * Marker interface for message handlers.
  *
  * @author Samuel Roze <samuel.roze@gmail.com>
+ *
+ * @deprecated since symfony/messenger 6.2, use the AsMessageHandler attribute instead
```

Comment:
perhaps add code like this also : 

https://github.com/symfony/symfony/blob/123b1651c4a7e219ba59074441badfac65525efe/src/Symfony/Component/DependencyInjection/Argument/ReferenceSetArgumentTrait.php#L14

Reply:
Triggering runtime deprecation notices from interfaces caused some issues in the past so let's try and see if tests keep passing. If it does break tests, then just keeping `@deprecated` is ok (DebugClassLoader will detect those and trigger a notice accordingly).

Reply:
oki I see thx for explanations

Reply:
Should be: `@deprecated since Symfony 6.2 [...]`
I agree with @94noni: this should trigger a runtime deprecation notice.
The codebase is not supposed to use the interface and this will ensure we do the needed work.
Tests that are relate to the interface should then be labeled with `@group legacy`.

Reply:
> The codebase is not supposed to use the interface

It does 🤷 
https://github.com/symfony/symfony/blob/7b212a11be00e080b2416e6f563cd5349df1a54f/src/Symfony/Component/Messenger/DependencyInjection/MessengerPass.php#L109
https://github.com/symfony/symfony/blob/7b212a11be00e080b2416e6f563cd5349df1a54f/src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php#L648

<img width="1027" alt="Screenshot 2022-09-25 at 19 20 24" src="https://user-images.githubusercontent.com/7502063/192157127-4dcbf02c-c7e5-4f19-aa0d-b863d28a0d69.png">

Added compile-time notices in MessengerPass instead and updated the tests accordingly.